src/main/java/com/epam/preprod/sirenko/FourElementsOfStringSumAsHashCodeWrapper.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.preprod.sirenko;
+
+/**
+ * String key wrapper, as HashCode uses sum of first four elements of string
+ *
+ * @author E.Sirenko
+ **/
+public class FourElementsOfStringSumAsHashCodeWrapper {
```

Comment:
Please create separate directory for stringWrappers and move them to it

Reply:
done